install.sh

```diff
@@ -38,6 +40,20 @@ ICU_ARCHIVE="icu4c-55_1-src.tgz"
 ICU_STANDARD=\
 "CXXFLAGS=-std=c++0x"
 
+# QRENCODE archive.
+#------------------------------------------------------------------------------
+QRENCODE_URL="http://fukuchi.org/works/qrencode/qrencode-3.4.4.tar.bz2"
+QRENCODE_ARCHIVE="qrencode-3.4.4.tar.bz2"
+QRENCODE_STANDARD=\
+"CXXFLAGS=-std=c++0x"
```

Comment:
Are `CXXFLAGS` here and below verified requirements of the build or are these cloned from the ICU settings?